autopilot/contract_pruning.go

```diff
@@ -105,15 +105,17 @@ func (ap *Autopilot) fetchHostContract(fcid types.FileContractID) (host api.Host
 	return
 }
 
-func (ap *Autopilot) performContractPruning(wp *workerPool) {
-	ap.logger.Info("performing contract pruning")
+func (ap *Autopilot) performContractPruning() {
+	log := ap.logger.Named("contractpruner")
```

Comment:
```suggestion
	log := ap.logger.Named("performContractPruning")
```

This is a bit more consistent with our logger naming outside of constructors. e.g. `performContractMaintenance` and `calculateMinScore`.